### ISSUE TITLE
Remove dead code in the css-multicol test suite

### ISSUE THREAD

**Issue by frivoal** [w3c/web-platform-tests#7469]
Some tests in the multicol test suite have dead code. It's confusing for anyone reading the test afterwards, and should be removed.

There's probably more, here's one instance I ran into:

* All the `span`, `span+span`, `span+span+span` styling in https://github.com/w3c/web-platform-tests/blob/5b857bf79ede0c99761aca3d5562e86ba979e49f/css/css-multicol-1/multicol-rule-shorthand-2.xht

**Comment by TalbotG** [comment]
Yes. There are tests with unrelated, irrelevant, pointless and/or useless declarations.

What most likely happened is that the original Opera author of those tests just copied and pasted a previous test and then only added a few declarations to it, therefore saving time. But that makes tests not streamlined, less compact and as you noted less clear.

**Comment by rachelandrew** [comment]
@gregwhitworth and I are currently doing a review of all the tests, so I'll identify stuff like this and deal with it once we get onto cleaning them up.

**Comment by TalbotG** [comment]
> There's probably more

Indeed

http://wpt.live/css/css-multicol/multicol-rule-px-001.xht 

http://wpt.live/css/css-multicol/multicol-shorthand-001.xht

Rachel,

In those 2, `span`, `span+span`, `span+span+span`, `orphans: 1` and `widows: 1` can be removed and should be removed.

**Comment by rachelandrew** [comment]
Thanks, I'm currently finishing off this review (I've been working through inlining these tests in the spec) I'll have a look at these.

**Comment by TalbotG** [comment]
Rachel, 

Some (several?) tests use `orphans: 1` and `widows: 1` when it is not justified, not needed, not useful to do so.

**Comment by TalbotG** [comment]
I am enclined to think that a **majority of tests using `orphans: 1` and `widows: 1`** should not have declared those. Back in 2013, when I reviewed many tests, I was not fully aware of how `orphans` and `widows` were working on multicol containers.

On July 24th 2013, I wondered about `orphans: 1` and `widows: 1` in tests:
https://lists.w3.org/Archives/Public/public-css-testsuite/2013Jul/0019.html
